coldfront/config/defaults.py

```diff
@@ -0,0 +1,155 @@
+"""Storage of default choices for attribute and status values."""
+
+### Allocation ###
+
+ALLOCATION_DEFAULTS = {
+    'attrtypes': (
+        'Date',
+        'Float',
+        'Int',
+        'Text',
+        'Yes/No',
```

Comment:
Just a side note, but I've been curious about why yes/no is used here instead of booleans and where the "no" option is more appropriate than "yes/no".